batchpr/updater.py

```diff
@@ -106,6 +106,7 @@ def run(self, repositories, delay=2):
             try:
                 print('  > Ensuring fork exists (and creating if not)')
                 self.ensure_fork_set_up()
+                time.sleep(30)
```

Comment:
maybe make this a kwarg, too?

Reply:
Any suggestion on the name? There is already a `delay` for something else. `fork_delay`?

Reply:
fork_delay sounds good

Reply:
OK done. Thanks for the review!